setup.py

```diff
@@ -16,6 +16,10 @@
     if requirements[0].startswith('-i'):
         requirements.pop(0)
 
+    print(requirements)
+
+    kwargs['extras_require']['aiida'] = requirements
```

Comment:
Wouldn't be many requirements duplicated in this way? Like `aiidalab-widgets-base` would appear here and in the basic aiidalab dependencies without extras.